### PR TITLE
[fix] StatsForecast - correct sampling frequency

### LINKED ISSUE #7193: [Bug]: Statsforecast - incorrect sampling frequency of the result set
### Short description of current behavior

Given some input data with a timestamp column sampled at 8 ms, the output returned by Statsforecast is sampled at 1 day (which is wrong), while the output returned by Lightwood is sampled at 8 ms (which is as expected).

### Video or screenshots

Using Lightwood results in correct output sampling:
image

Using Statsforecast results in incorrect output sampling:
image

### Expected behavior

Statsforecast should sample the output data to match the input data sampling frequency.

### How to reproduce the error

Follow this tutorial: https://docs.mindsdb.com/sql/tutorials/eeg-forecasting

Lightwood samples output at 8 ms, matching the input data sampling frequency, but Statsforecast samples the output every 1 day, instead of every 8 ms.

Using Lightwood results in correct output sampling:
image

Using Statsforecast results in incorrect output sampling:
image

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by paxcema**
## Description

Fixes #7193, again.

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: mindsdb/integrations/handlers/statsforecast_handler/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-statsforecast>=1.4.0, <2.0
```
Comment (paxcema) [inline mindsdb/integrations/handlers/statsforecast_handler/requirements.txt:1]: Pinning for stability

Comment (pedrofluxa) [reply mindsdb/integrations/handlers/statsforecast_handler/requirements.txt:1]: Agreed.

**Review by pedrofluxa — APPROVED** [review]
Looks good. Approved.

**Comment by AndrewFarley** [comment]
@paxcema This breaks compatibility with installing lightwood.  Surely a soft version pin is better and gives pip some wiggle room.  Currently it is impossible to install mindsdb + lightwood on the same machine because of this.

https://github.com/mindsdb/lightwood/blob/staging/requirements.txt#L21C1-L21C14
```
lightwood ==1.4.0
```

**Pull request merged**